src/DocumentFormat.OpenXml.Framework/Equality/HashCode.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace DocumentFormat.OpenXml
+{
+    internal ref struct HashCode
+    {
```

Comment:
There's a shim for this already in the common directory and brought in via https://github.com/dotnet/Open-XML-SDK/blob/main/src/common/FrameworkShims.targets

Reply:
It seems to me that the DocumentFormat.OpenXml project does not include these:

See [here](https://github.com/dotnet/Open-XML-SDK/blob/e7a0331218de7f53582698bdfd1ddcc2124c1a40/src/DocumentFormat.OpenXml/DocumentFormat.OpenXml.csproj#L8C48-L8C48)

Reply:
Hmm then for now include it manually and I'll clean it up